### PR TITLE
Dispatch to InputEvents from patchwork-god-classes

### PR DESCRIPTION
**Pull request by kitlith**
Depends on #127, can rebase, etc., once that gets in. Until then, please ignore the first two commits that are part of #127.

### CODE REVIEW

**Comment by kitlith** [comment]
rebase'd w/ `git rebase --onto master event-dispatcher-module`

File: patchwork-events-input/src/main/java/net/patchworkmc/mixin/event/input/MixinMouse.java [hunk 1]
```diff
@@ -32,36 +28,26 @@
...
-	abstract boolean wasRightButtonClicked();
-	@Shadow
-	abstract double getX();
-	@Shadow
-	abstract double getY();
+import net.patchworkmc.api.input.ForgeMouse;
+import net.patchworkmc.impl.event.input.InputEvents;
 
+@Mixin(Mouse.class)
+public abstract class MixinMouse implements ForgeMouse {
```
Comment (TheGlitch76) [inline patchwork-events-input/src/main/java/net/patchworkmc/mixin/event/input/MixinMouse.java:35]: This doesn't actually implement the interface?

Comment (kitlith) [reply patchwork-events-input/src/main/java/net/patchworkmc/mixin/event/input/MixinMouse.java:35]: huh. whoops. i must've gotten it in place and then forgot to actually implement it.

**Comment by kitlith** [comment]
rebased

**Pull request merged**